container/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# Courtesy of Federico Ficarelli
+
+FROM ghcr.io/pulp-platform/snitch_cluster:latest@sha256:9978f9d0e3e6b7ad1b2c1f0eb628a113226e8443bc42ae8b4ddd3f34d9528e96 as builder
+
+RUN git clone --recursive https://github.com/kuleuven-micas/snitch_cluster.git /src \
+ # verilator model
+ && cd /src/target/snitch_cluster \
+ && make bin/snitch_cluster.vlt
```

Comment:
Just a thought, but what if we have different builds? I mean, what if we can reconfigure the .hjson file to different architectures. With SNAX, no SNAX, 1 SNAX 1 Snitch and 1 DMA and so on. I wonder if it's possible if we can have a switcher for that. Right now, it's not really needed but I foresee we will need that in the future.

Reply:
Good call, I can add a docker build arg there.
https://vsupalov.com/docker-env-vars/

Reply:
Added in [3d830ef](https://github.com/KULeuven-MICAS/snax-mlir/pull/2/commits/3d830eff8e1d08d60b34b98db4108399156ed987)